image-overview-templates/new-names/security-proxy-auth.md

```diff
@@ -0,0 +1,19 @@
+$$./common/warning.md
+
+$$./common/quick-reference.md
+
+# Supported tags and respective Dockerfile links
+
+$$./common/version-minnesota.md |
+        - [Dockerfile](https://github.com/edgexfoundry/edgex-go/blob/v3.0.0/cmd/security-proxy-auth/Dockerfile)
+
+$$./common/quick-reference-cont-edgex-go.md
+
+$$./common/what-is-edgex.md
+
+# What's in this image?
+
```

Comment:
@bnevis-i , please provide short description of this service image

i.e. this image contains the service that ....

Reply:
"This image was added in EdgeX 3.0 to perform delegated authentication for the NGINX API gateway.  It evaluates the incoming JWT on API gateway calls and returns success (HTTP 200) or failure (HTTP 401) to authorize or deny the request from being forwarded to the backend EdgeX microservice."